lib/ts/framework/request.ts

```diff
@@ -63,4 +63,26 @@ export abstract class BaseRequest {
         }
         return this.parsedJSONBody;
     };
+
+    getBodyAsJSONOrFormData = async (): Promise<any> => {
+        const contentType = this.getHeaderValue("content-type") || this.getHeaderValue("Content-Type");
```

Comment:
I think the casing should be normalized by the `getHeaderValue`.

Reply:
done